eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  SuppressionFilePath: 'eng\CredScanSuppression.json'
+  BaselineFilePath: ''
+steps:
+- pwsh: |
+    New-Item -Path "$(Build.sourcesdirectory)" -Name "credscan.tsv"
+    if ("$(Build.Reason)" -eq 'PullRequest') {
+      New-Item -Path "$(Build.sourcesdirectory)" -Name "changedFiles" -ItemType "directory"
+      (git diff "origin/$(System.PullRequest.TargetBranch)" HEAD --name-only)
```

Comment:
If this change goes in we should be sure to add it to the list in issue https://github.com/Azure/azure-sdk-tools/issues/2531 of places we need to consolidate so that we have one common way to produce the list of changed files.

Reply:
added the PR into the issue.